src/vm/wren_core.wren

```diff
@@ -323,6 +323,43 @@ class List is Sequence {
     return other
   }
 
+  sort() {
+    sort {|a, b| a < b }
+    return this
```

Comment:
I think it's rather weird for `sort()` to return `this` but not `sort(comparer)`. I'd suggest changing this back to the one-line version and putting the `return this` in `sort(comparer)`.